phpunit/phpunit/4.7/phpunit.xml.dist

```diff
@@ -5,16 +5,16 @@
          xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.1/phpunit.xsd"
          backupGlobals="false"
          colors="true"
-         bootstrap="vendor/autoload.php"
+         bootstrap="tests/bootstrap.php"
 >
     <php>
         <ini name="error_reporting" value="-1" />
         <env name="KERNEL_CLASS" value="App\Kernel" />
+        <env name="SHELL_VERBOSITY" value="-1" />
+
+        <!-- override or set env variables for the test env here -->
         <env name="APP_ENV" value="test" />
         <env name="APP_DEBUG" value="1" />
```

Comment:
I don't believe this even works. This will be a string "1". I think we should remove it.